src/components/OperationStatus.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "@canonical/react-components";
+import { useOperations } from "context/operationsProvider";
+import React, { useState } from "react";
+import { isWidthBelow } from "util/helpers";
+import useEventListener from "@use-it/event-listener";
+import { Link } from "react-router-dom";
+import { pluralize } from "util/instanceBulkActions";
+
+const isSmallScreen = () => isWidthBelow(620);
+
+const OperationStatus = () => {
+  const [reduceText, setReduceText] = useState(false);
```

Comment:
Better call the state vars `isSmallScreen`?  I didn't get initially what `reduceText` is used for. We can directly call isWidthBelow in line 15 maybe to avoid duplicating the name?

Reply:
yeah that makes sense